app/views/claims/schools/users/check.html.erb

```diff
@@ -0,0 +1,91 @@
+<%= content_for :page_title, sanitize(t(".page_title", school_name: @school.name)) %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(
+    href: new_claims_school_user_path(
+      @user_form.as_form_params.merge(school_id: @school.id),
+    ),
+  ) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_with(model: @user_form, url: claims_school_users_path, method: :post) do |f| %>
+        <%= f.hidden_field :first_name, value: @user_form.first_name %>
+        <%= f.hidden_field :last_name, value: @user_form.last_name %>
+        <%= f.hidden_field :email, value: @user_form.email %>
+
+        <label class="govuk-label govuk-label--l">
+          <span class="govuk-caption-l"><%= t(".caption", school_name: @school.name) %></span>
+          <%= t(".title") %>
+        </label>
+
+        <dl class="govuk-summary-list">
```

Comment:
could we place use the govuk_summary_list component here? See [here](https://github.com/DFE-Digital/itt-mentor-services/blob/ec5fcd1c683c9e5373473a93718bb3204e10b851/app/views/placements/support/organisations/users/_check_form.html.erb
)
Better yet, move that component to a shared space and use it directly so we don't have so much repetition.